pkg/ingest/testkafka/cluster.go

```diff
@@ -70,7 +70,7 @@ func (c *Cluster) ensureConsumerGroupExists(consumerGroup string) {
 	}
 	c.committedOffsets[consumerGroup] = make([]int64, c.numPartitions+1)
 
-	// Initialise the partition offsets with the special value -1 which means "no offset committed".
+	// Initialise the Partition offsets with the special value -1 which means "no offset committed".
```

Comment:
nit: Is this capitalization intentional?

Reply:
Fixed